Snapshot config can be set in schema / properties YAML file starting v1.9 (#6206)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## October 2024

> - **Enhancement**: In dbt Cloud Versionless, you can now specify snapshot configurations in YAML files, including `schema.yml` or `dbt_project.yml`, for snapshots defined in SQL files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in the SQL file. This enhancement will soon be released in the upcoming dbt Core v1.9 release

Reviewer: @dbeatty10 added a release note entry for this - can yo triple check

Author: ```suggestion
- **Enhancement**: In dbt Cloud Versionless, snapshots defined in SQL files can now use `config` defined in `schema.yml` YAML files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in `dbt_project.yml` and/or a `config()` block within the SQL file. This enhancement will be included in the upcoming dbt Core v1.9 release.
```
